src/main/java/uk/gov/laa/ccms/caab/controller/application/summary/EditProceedingsAndCostsSectionController.java

```diff
@@ -25,63 +61,679 @@
+        .orElse(Collections.emptyList());
+
+    model.addAttribute("proceedingTypes", proceedingDetails);
+  }
+
+  /**
+   * Handles the GET request to display the further details of a proceeding.
+   *
+   * @param proceedingFlow The proceeding flow data.
+   * @param application The application details.
+   * @param model The Model object to add attributes to for the view.
+   * @return The name of the view to be rendered.
+   */
+  @GetMapping("/application/summary/proceedings/{action}/further-details")
+  public String proceedingsActionFurtherDetails(
+      @SessionAttribute(PROCEEDING_FLOW_FORM_DATA) final ProceedingFlowFormData proceedingFlow,
+      @SessionAttribute(APPLICATION) final ApplicationDetail application,
+      final Model model) {
+
+    populateFurtherDetailsDropdowns(model,
+        application,
+        proceedingFlow.getProceedingDetails(),
+        proceedingFlow.getMatterTypeDetails().getMatterType());
+
+    model.addAttribute(PROCEEDING_FLOW_FORM_DATA, proceedingFlow);
+    model.addAttribute("furtherDetails",
+        proceedingFlow.getFurtherDetails());
+
+    return "application/summary/proceedings-further-details";
+  }
+
+  /**
+   * Handles the POST request to update the further details of a proceeding.
+   *
+   * @param application The application details.
+   * @param proceedingFlow The proceeding flow data.
+   * @param furtherDetails The further details of the proceeding.
+   * @param action The action being performed (add or edit).
+   * @param bindingResult The results of validation.
+   * @param model The Model object to add attributes to for the view.
+   * @param session The HTTP session.
+   * @return A redirect instruction to the proceedings confirm view.
+   */
+  @PostMapping("/application/summary/proceedings/{action}/further-details")
+  public String proceedingsActionFurtherDetailsPost(
+      @SessionAttribute(APPLICATION) final ApplicationDetail application,
+      @SessionAttribute(PROCEEDING_FLOW_FORM_DATA) final ProceedingFlowFormData proceedingFlow,
+      @ModelAttribute("furtherDetails") final ProceedingFormDataFurtherDetails furtherDetails,
+      @PathVariable("action") final String action,
+      final BindingResult bindingResult,
+      final Model model,
+      final HttpSession session) {
+
+
+    if (action.equals(ACTION_ADD)) {
+
+      proceedingFlow.setFurtherDetails(furtherDetails);
+      furtherDetailsValidator.validate(proceedingFlow, bindingResult);
+
+      if (bindingResult.hasErrors()) {
+        populateFurtherDetailsDropdowns(model,
+            application,
+            proceedingFlow.getProceedingDetails(),
+            proceedingFlow.getMatterTypeDetails().getMatterType());
+        model.addAttribute(PROCEEDING_FLOW_FORM_DATA, proceedingFlow);
+        return "application/summary/proceedings-further-details";
+      }
+
+      proceedingFlow.setAmended(Boolean.TRUE);
+    } else {
+
+      final ProceedingFlowFormData oldProceedingFlow =
```

Comment:
Does the proceedingFlow not need to be validated for an edit?

Reply:
Amended the if statement and moved validation before if